miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -204,24 +205,35 @@ abstract class MiniApp internal constructor() {
                 registerApiClient(defaultConfig.key, apiClient)
             }
 
+            val signatureVerifier: SignatureVerifier? = SignatureVerifier.init(
+                context = context,
+                baseUrl = miniAppSdkConfig.baseUrl + "keys/",
```

Comment:
If we use retrofit and the same `ApiClient` in the miniapp sdk , we can avoid sending `"keys/"` path as a base url, we can simple add this path in the api call..